### ISSUE TITLE
Vulnerabilities when running npn i  as instructed on week 0

### ISSUE THREAD

**Issue by rcontesti** [ExamProCo/aws-bootcamp-cruddur-2023#23]
Hi, first of all, thank so much for sharing all this knowledge. I'm late to the boat and when tried to run `npn i` in gitpod, I'm not able to fix the issues appearing. Curious if someone else is running into the same problem. Thank you!

`131 vulnerabilities (1 low, 104 moderate, 22 high, 4 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force`

**Comment by Ismail-khlif** [comment]
@rcontesti  i think it's okay , try npm i --f  , but my question for you is there a docker-compose.yml ?

**Comment by rcontesti** [comment]
> @rcontesti i think it's okay , try npm i --f , but my question for you is there a docker-compose.yml ?

@Ismail-khlif, yes I try it but still I get several vulnerabilities. Regarding docker-compose.yml, I cannot find any within the folders. Is there any folder where I should I find it?  

Many thanks!
